src/web/traceViewer/ui/workbench.tsx

```diff
@@ -89,10 +91,10 @@ export const Workbench: React.FunctionComponent<{
       <SplitView sidebarSize={300} orientation='horizontal'>
         <SnapshotTab action={selectedAction} snapshotSize={snapshotSize} />
         <TabbedPane tabs={[
-          { id: 'logs', title: 'Log', render: () => <LogsTab action={selectedAction} /> },
-          { id: 'console', title: 'Console', render: () => <ConsoleTab action={selectedAction} /> },
-          { id: 'source', title: 'Source', render: () => <SourceTab action={selectedAction} /> },
-          { id: 'network', title: 'Network', render: () => <NetworkTab action={selectedAction} /> },
+          { id: 'logs', title: 'Call', count: 0, render: () => <CallTab action={selectedAction} /> },
```

Comment:
Maybe 'Action' instead of 'Call'? For me, call without any adjective means "phone call" :)

Reply:
It is a method call :) innerText() is not really an action...